src/zconf.h

```diff
@@ -1,5 +1,5 @@
 /* zconf.h -- configuration of the zlib compression library
- * Copyright (C) 1995-2004 Jean-loup Gailly.
+ * Copyright (C) 1995-2004 Jean-Loup Gailly.
```

Comment:
Just today I came across a reference that this routine is likely one of the three most-widely-used pieces of code in trillions of instances across the globe.

In each and every copyright header Mr Gailly writes his name as 'Jean-loup'. I think we should let him.

Reply:
I didn't know that he spelled his name that way. The only reason I made the change is because it was inconsistently capitalized in other places. I agree that anyone should get to spell their own name the way that they like. I've reverted throughout where I made it upper-case, and I made it lower case where it was upper case in a few places.